Cargo.toml

```diff
@@ -35,8 +35,9 @@ serde_derive = "1.0"
 serde_json = "1.0"
 rand = "0.8"
 rand_distr = "0.4"
-remoteprocess = {version="0.4.12", features=["unwind"]}
+remoteprocess = { git = "https://github.com/akhramov/remoteprocess", branch = "feature/windows-aarch64" }
```

Comment:
to-do: Unrelated to this effort. Remove this, needed on my machine only, to be upstreamed in https://github.com/benfred/remoteprocess/pull/89